xdebug/xdebug.php

```diff
@@ -351,7 +352,7 @@ function xdebug_get_gc_total_collected_roots(): int {}
  * @param int   $listType
  * @param array $configuration
```

Comment:
I think it makes sense to update the return tag in phpdoc according to function signature